### ISSUE TITLE
Error reading Proxy certificates. (tlsv1 alert unknown ca)

### ISSUE THREAD

**Issue by micafer** [requests/requests#4630]
Error reading Proxy certificates. It works for requests versions 2.11.1 and older.
But it does not work for 2.12.0 and newer versions.

## Expected Result

```
<Response [200]>
```

## Actual Result
```
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='cloud.recas.ba.infn.it', port=5000): Max retries exceeded with url: /v2.0/tokens (Caused by SSLError(SSLError("bad handshake: Error([('SSL routines', 'ssl3_read_bytes', 'tlsv1 alert unknown ca')],)",),))
```

## Reproduction Steps

```python
import requests

body = '{"auth":{"voms":true}}'
headers = {'Accept': 'application/json', 'Connection': 'close', 'Content-Type': 'application/json'}
url = "https://<keystone_server>:5000/v2.0/tokens"
# it is a grid proxy certificate
cert = "/tmp/proxy.pem" # or cert = ("/tmp/proxy.pem", "/tmp/proxy.pem") 
requests.request('POST', url, verify=False, cert=cert, headers=headers, data=body)
```

## System Information

    $ python -m requests.help

```
{
  "chardet": {
    "version": "3.0.4"
  }, 
  "cryptography": {
    "version": "2.2.2"
  }, 
  "idna": {
    "version": "2.6"
  }, 
  "implementation": {
    "name": "CPython", 
    "version": "2.7.13"
  }, 
  "platform": {
    "release": "4.4.0-121-generic", 
    "system": "Linux"
  }, 
  "pyOpenSSL": {
    "openssl_version": "1010008f", 
    "version": "17.5.0"
  }, 
  "requests": {
    "version": "2.18.4"
  }, 
  "system_ssl": {
    "version": "1010006f"
  }, 
  "urllib3": {
    "version": "1.22"
  }, 
  "using_pyopenssl": true
}
```

**Comment by sigmavirus24** [comment]
The primary difference that looks potentially relevant to this [between 2.11.1 and 2.12.0](https://github.com/requests/requests/compare/v2.11.1...v2.12.0) is in urllib3. I believe that if the certificate authority file is false, we attempt to load the default file for the operating system. It seems that the code in [question](https://github.com/urllib3/urllib3/blob/30e9855307aaa9f9f09651f2bbddeb7350eed666/urllib3/util/ssl_.py#L334..L347) doesn't check the `cert_reqs` parameter which we [set to bypass verification](https://github.com/urllib3/urllib3/blob/30e9855307aaa9f9f09651f2bbddeb7350eed666/urllib3/util/ssl_.py#L334..L347). It also seems plausible to me that `load_default_certs` is resetting `cert_reqs` so that we do try to verify the TLS certificate when connecting.

Either way, this needs further investigation in urllib3.

**Comment by micafer** [comment]
This isssue is solved in urllib 1.23.
And it seems that you have updated you code to use update the urllib3 version in this commit
https://github.com/requests/requests/commit/ac944b7439009ffbf7a10dfee35202f1ac090e76.
I'm waiting it to be released.
